features/liquidity/components/ManageLiquidityCard.tsx

```diff
@@ -13,16 +12,16 @@ import {
   Text,
   useSubscribeInteractions,
 } from 'junoblocks'
+import { PoolState, PoolTokenValue, ReserveType } from 'queries/useQueryPools'
 
 import { UnderlyingAssetRow } from './UnderlyingAssetRow'
 
-type ManageLiquidityCardProps = Pick<
-  LiquidityInfoType,
-  | 'myLiquidityReserve'
-  | 'myStakedLiquidityReserve'
-  | 'tokenDollarValue'
-  | 'myStakedLiquidity'
-> & {
+type ManageLiquidityCardProps = {
+  stakedLiquidity: PoolState
+  providedLiquidity: PoolTokenValue
+  providedTotalLiquidity: PoolTokenValue
+  providedLiquidityReserve: ReserveType
```

Comment:
what is the difference between provided liquidity and provided liquidity reserve?

Reply:
`providedLiquidityReserve` is an array with 2 numbers. One is for the first token pair in the pool and the other one is for the second token. `providedLiquidity` contains full lp tokenAmount the user provided in the pool and what it is in dollar value